Mighty-Menorahs/reactive-nutshell
# got add message form to work and got delete function working

# Description

You now be able to add a new message and see the name of the sender as well as the time stamp of the message. You can also delete the message.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ x] New feature (non-breaking change which adds functionality)

# Testing Instructions

git fetch --all
git checkout cj-message-form

You now be able to add a new message and see the name of the sender as well as the time stamp of the message. You can also delete the message.


# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
